scripts/lisk_bridge.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Lisk bridge
+# Connects source and target versions of Lisk
```

Comment:
License header:

```
# LiskHQ/lisk-bridge
#
# Connects source and target versions of Lisk in order to migrate
# gracefully between protocol changes.
#
# Copyright (C) 2017 Lisk Foundation
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.
######################################################################
```